songs/package.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "adiherzog.github.io",
+    "version": "0.0.2",
+    "description": "",
+    "main": "index.html",
+    "scripts": {
+        "test": "eslint './**/*.js'"
+    },
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/xeronimus/adiherzog.github.io.git"
```

Comment:
this is a file generated by npm itself.  the url format is therefore suggested by npm.. should we change it anyway`?